Exercises/21/index.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
+
+alert("Welcome to Lesson 21");
+let firstNumber = prompt("Enter first number", 0);
+let secondNumber = prompt("Enter the second number", 0);
```

Comment:
As a best practice, convert the values to numbers before comparing. Good job though @clopez53 💯